Improve documentation (#321)
kimai/www.kimai.org: _documentation/emails.md

### MAILER_URL

> The following list of providers is supported out-of-the-box by Kimai: 

Reviewer: ```suggestion
The following providers are supported out-of-the-box by Kimai: 
```

- Deactivated: `null://null`

> - SMTP: `smtp://localhost:25?encryption=&auth_mode=`

Reviewer: Outdated example
```suggestion
- SMTP: `smtp://localhost:25`
```

Author: this is already fixed in another PR.
